shell/components/BrandImage.vue

```diff
@@ -49,7 +53,7 @@ export default {
     },
 
     pathToBrandedImage() {
-      if (this.fileName === 'rancher-logo.svg') {
+      if (this.fileName === 'rancher-logo.svg' || this.supportCustomLogo) {
```

Comment:
Why is this needed instead of using the `brand` mechanism?

Reply:
Because we need to support custom logos at the top of the menu, such as this
<img width="1715" alt="image" src="https://github.com/rancher/dashboard/assets/24985926/f176f0eb-f3e9-4c47-800e-99d45aca8418">